mappings/net/minecraft/client/texturepack/TexturePackManager.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_573 net/minecraft/client/texturepack/TexturePackManager
+	FIELD field_2129 skin Lnet/minecraft/class_675;
+	FIELD field_2131 skin2 Lnet/minecraft/class_675;
+	FIELD field_2133 client Lnet/minecraft/client/Minecraft;
+	FIELD field_2134 texturepacksDir Ljava/io/File;
```

Comment:
texturepacks -> texturePacks?

Reply:
Hmm, while you could make this change i have spotted a fair few other ones without the capital P. this is okay.